src/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Net.Security.Tests
 {
     internal static class TestConfiguration
     {
-        public const int PassingTestTimeoutMilliseconds = 1 * 60 * 1000;
+        public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
```

Comment:
Ok.  Let's just remember that this was us increasing the value to practically infinity, so that in a few months we don't increase it again if/when the failures occur and we think "hmm, I wonder if they're legitimately timing out or hanging, let's increase it to see" ;)